src/ceph.in

```diff
@@ -836,7 +836,7 @@ def get_scrub_timestamps(childargs):
     devnull = open(os.devnull, 'w')
     out = subprocess.check_output(['ceph', 'pg', 'dump', '--format=json-pretty'],
                                   stderr=devnull)
-    for stat in json.loads(out)['pg_stats']:
+    for stat in json.loads(out)['pg_map']['pg_stats']:
```

Comment:
This will also need to handle the old format (catch KeyError, etc), as our tools are backward compatible by two major versions.

Reply:
updated it.
thanks